aggregators/internal/protohash/generate/main.go

```diff
@@ -0,0 +1,118 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package main
+
+import (
+	"fmt"
+	"go/types"
+	"log"
+	"os"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+)
+
+func main() {
+	const pkgpath = "github.com/elastic/apm-aggregation/aggregationpb"
+	cfg := &packages.Config{Mode: packages.NeedTypes | packages.NeedTypesInfo}
+	pkgs, err := packages.Load(cfg, pkgpath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	f, err := os.Create("generated.go")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer f.Close()
+
+	fmt.Fprintln(f, `
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+// Code generated by protohash/generate. DO NOT EDIT.
+
+package protohash
```

Comment:
Adding a package comment here should fix the lint complaint.

Reply:
There's a package doc comment in doc.go, I just misplaced it - fixed now.